deploy/docker-compose/configs/informatics-gateway.json

```diff
@@ -0,0 +1,117 @@
+{
+  "ConnectionStrings": {
+    "InformaticsGatewayDatabase": "Data Source=/database/mig.db"
+  },
+  "InformaticsGateway": {
+    "dicom": {
+      "scp": {
+        "port": 104,
+        "logDimseDatasets": false,
+        "rejectUnknownSources": true
```

Comment:
I have the opinion of NOT blocking unknown sources by default. Given the intended use, this will avoid failures on user error setting up the external PACS or the IG config. Give a warning in the doc though, any junk DICOM Device can send to it.